llm_request.py

```diff
@@ -209,10 +209,18 @@ def make_openai_url_and_headers(ctx: ApiContext, path: str):
     url = ctx.base_url or "https://api.openai.com/v1"
     hostname = urllib.parse.urlparse(url).hostname
     use_azure_openai = hostname and hostname.endswith("openai.azure.com")
+    use_ovh = hostname and hostname.endswith("cloud.ovh.net")
```

Comment:
Is this necessary? The default case will set content-type and authorization headers already.

Reply:
Thank you Justin, I get your point but yes it is necessary for the moment. The key you are going to use requires a specific format for the Authorization header on our end. It is temporary, I'll make a new PR to remove this specific behavior as soon as it is possible.

Reply:
I see. Is it the lack of "Bearer" that you need? Or lowercase "authorization"?

Reply:
The lack of "Bearer".